modules/gdnative/gdnative/color.cpp

```diff
@@ -31,178 +31,17 @@
-}
-
-godot_real godot_color_get_r(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->r;
-}
-
-void godot_color_set_r(godot_color *p_self, const godot_real val) {
-	Color *self = (Color *)p_self;
-	self->r = val;
-}
-
-godot_real godot_color_get_g(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->g;
-}
-
-void godot_color_set_g(godot_color *p_self, const godot_real val) {
-	Color *self = (Color *)p_self;
-	self->g = val;
-}
-
-godot_real godot_color_get_b(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->b;
-}
-
-void godot_color_set_b(godot_color *p_self, const godot_real val) {
-	Color *self = (Color *)p_self;
-	self->b = val;
-}
-
-godot_real godot_color_get_a(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->a;
-}
-
-void godot_color_set_a(godot_color *p_self, const godot_real val) {
-	Color *self = (Color *)p_self;
-	self->a = val;
-}
-
-godot_real godot_color_get_h(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->get_h();
-}
-
-godot_real godot_color_get_s(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->get_s();
-}
-
-godot_real godot_color_get_v(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->get_v();
-}
-
-godot_string GDAPI godot_color_as_string(const godot_color *p_self) {
-	godot_string ret;
-	const Color *self = (const Color *)p_self;
-	memnew_placement(&ret, String(*self));
-	return ret;
-}
-
-godot_int GDAPI godot_color_to_rgba32(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_rgba32();
-}
-
-godot_int GDAPI godot_color_to_abgr32(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_abgr32();
-}
-
-godot_int GDAPI godot_color_to_abgr64(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_abgr64();
-}
-
-godot_int GDAPI godot_color_to_argb64(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_argb64();
-}
-
-godot_int GDAPI godot_color_to_rgba64(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_rgba64();
-}
-
-godot_int GDAPI godot_color_to_argb32(const godot_color *p_self) {
-	const Color *self = (const Color *)p_self;
-	return self->to_argb32();
-}
-
-godot_color GDAPI godot_color_inverted(const godot_color *p_self) {
-	godot_color dest;
-	const Color *self = (const Color *)p_self;
-	*((Color *)&dest) = self->inverted();
-	return dest;
-}
-
-godot_color GDAPI godot_color_lerp(const godot_color *p_self, const godot_color *p_b, const godot_real p_t) {
-	godot_color dest;
-	const Color *self = (const Color *)p_self;
-	const Color *b = (const Color *)p_b;
-	*((Color *)&dest) = self->lerp(*b, p_t);
-	return dest;
-}
+#include <gdnative/gdnative.h>
```

Comment:
You didn't add this for other classes, is this unused?

Reply:
Yeah, copy-paste mistake. I double-checked it but this one still slipped through.